src/cli/self_update.rs

```diff
@@ -622,6 +621,7 @@ fn install_bins() -> Result<()> {
 }
 
 pub fn install_proxies() -> Result<()> {
+    use same_file::Handle;
```

Comment:
I don't think this change is worth making.

Reply:
Done.